python/python/Ice/BatchRequestInterceptor.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
My understanding is that Python 3 is utf8 by default and so this is no longer needed. Not having it would allow for a cleaner header as well.

Reply:
Sounds good, but can wait until a follow-up PR.